Core/IntegrationTests/ServerTransportTests.cs

```diff
@@ -69,7 +69,7 @@ public async Task SendAndReceiveObjectWithBlobs()
     var myObject = Fixtures.GenerateSimpleObject();
     myObject["blobs"] = Fixtures.GenerateThreeBlobs();
 
-    var sentObjectId = await Operations.Send(myObject, new List<ITransport> { transport }).ConfigureAwait(false);
+    var sentObjectId = await Operations.Send(myObject, transport, false).ConfigureAwait(false);
```

Comment:
Bit concerned here that we were previously using the default SQLite transport... If so, that wouldn't actually test our server transport.

Am I ok to assume this wasn't the intention? @gjedlicska , and I'm right to change this test?

Reply:
Gergo confirmed that the new behaviour is correct (potentially before we were relying on cache missess to properly cover)